package/kedro_viz/api/rest/responses.py

```diff
@@ -419,12 +419,12 @@ def save_api_main_response_to_fs(main_path: str, remote_fs: Any):
         raise exc
 
 
-def save_api_node_response_to_fs(nodes_path: str, remote_fs: Any):
+def save_api_node_response_to_fs(nodes_path: str, remote_fs: Any, preview: bool):
```

Comment:
same comment as above for naming - `is_preview_enabled`

Reply:
Done.